internal/core/config/config.go

```diff
@@ -22,6 +22,11 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const (
+	protoSAPM = "sapm"
+	protoSFX  = "sfx"
```

Comment:
The protocol is actually Zipkin v2.  My understanding is that we are trying to avoid calling it a "SignalFx" protocol.

Reply:
Updated